### PR TITLE
add Dutch P&C FC model info

### PR DESCRIPTION
**Pull request by zhehuaichen**
# What does this PR do ?

Update scores table for Fast Conformer Dutch models

**Collection**: [ASR]

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation

### CODE REVIEW

**Comment by nithinraok** [comment]
jenkins

File: docs/source/asr/scores.rst [hunk 1]
```diff
@@ -349,6 +349,15 @@ ZH
     
 --------------------
 
+NL
```
Comment (nithinraok) [inline docs/source/asr/scores.rst:352]: @zhehuaichen can you place this subsection in the order of alphabetical (except En) same for PnC section.

Comment (nithinraok) [inline docs/source/asr/scores.rst:352]: @zhehuaichen  can you place this subsection in the order of alphabetical (except En) same for PnC section.

**Review by nithinraok — CHANGES_REQUESTED** [review]

File: docs/source/asr/scores.rst (hunk 1 above)
Comment (zhehuaichen) [reply docs/source/asr/scores.rst:352]: done

**Review by nithinraok — APPROVED** [review]
LGTM Thanks.

**Comment by nithinraok** [comment]
jenkins

**Pull request merged**